models/src/test/java/io/sphere/sdk/reviews/commands/updateCommands/SetAuthorNameTest.java

```diff
@@ -0,0 +1,20 @@
+package io.sphere.sdk.reviews.commands.updateCommands;
+
+import org.junit.Test;
+
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by njacinto on 21/04/15.
+ */
+public class SetAuthorNameTest {
```

Comment:
This class is implicitly tested with the ReviewUpdateCommand test. So we can get rid of this test.